libc/CMakeLists.txt

```diff
@@ -60,6 +60,10 @@ if(LLVM_LIBC_FULL_BUILD OR LLVM_LIBC_GPU_BUILD)
     message(STATUS "Will use ${LIBC_HDRGEN_EXE} for libc header generation.")
   endif()
 endif()
+# We will build the GPU utilities if we are not doing a runtimes build.
+if((LIBC_TARGET_OS_IS_GPU OR LLVM_LIBC_GPU_BUILD) AND NOT LLVM_RUNTIMES_BUILD)
```

Comment:
Why do you have to skip this when `LLVM_RUNTIMES_BUILD` is set?

Reply:
Because in a full cross-compiling build we don't necessarily have access to the LLVM and CUDA libraries that it requires. I already have around `find_package(CUDA)` and `find_package(LLVM)` but it wasn't a sound solution. This is pretty much the same reason we don't build `libc-hdrgen` during the runtimes build.